src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -173,18 +172,18 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     config = loadConfig(config_filename)
     num_tests = len(config)
     test_pass_threshold = 0.999 #percentage of passed test expected
-    num_passed_tests = 0.0
+    num_passed_tests = 1
```

Comment:
Why are you starting this (and the others below) at 1?

Reply:
Error... fixed